proxy.php

```diff
@@ -0,0 +1,267 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ * Piwik Proxy Hide URL
+ *
+ * @link http://piwik.org/faq/how-to/#faq_132
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
```

Comment:
can we maybe in the other two files, define a constant, and then do something like this?

```php
if (!defined( 'MYVARIABLENAME')) {
    exit; // this file is not supposed to be accessed directly
}
```

Reply:
Will do.